indico/MaKaC/webinterface/tpls/RoomBookingAdminLocation.tpl

```diff
@@ -386,7 +386,7 @@ indicoRequest(
               <span style="background-color: #C9FFC9; font-weight: bold;">
                 ${ '{0:.02f}'.format(kpi['occupancy'] * 100) }<small>%</small>
               </span>
-              ${inlineContextHelp('Average room occupancy in last 30 days during working hours (8H30-17H30, Monday-Friday including holidays). Only active, publically reservable rooms are taken into account.' )}
+              ${inlineContextHelp('Average room occupancy in last 30 days during working hours (8:30 - 12:30 and 13:30 - 17:30, Monday-Friday including holidays). Only active, publically reservable rooms are taken into account.' )}
```

Comment:
Thought of it, but then I will have to take care of i18n and the whole thing will end up being a mess. When we turn this into a true setting, then it makes sense to make it dynamic, IMHO.